packages/material-ui-styled-engine-sc/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "5.0.0-alpha.22",
   "private": false,
   "author": "Material-UI Team",
-  "description": "Material-UI Styled - Wrapper package for styled-components.",
+  "description": "Styled wrapper package for styled-components.",
```

Comment:
What does "Styled" refer to here?

Reply:
"Styled" refers to this API: https://emotion.sh/docs/styled.

Reply:
So:
```suggestion
  "description": "`@emotion/styled wrapper package for styled-components.",
```
?

Reply:
Not sure, we are diverging from emotion and sc to find the common denominator.

Reply:
So does it refer to that API or not? 🤷🏼‍♂️

Reply:
It refers to the general pattern:

const StyledDiv = styled('div')({ color: 'red' });

Reply:
```suggestion
  "description": "Styled API wrapper package for styled-components.",
```